packages/ui/components/pages/Fuse/FusePoolPage/MarketsList/AdditionalInfo.tsx

```diff
@@ -0,0 +1,239 @@
+import { FundOperationMode } from '@midas-capital/types';
+import { Row } from '@tanstack/react-table';
+import { utils } from 'ethers';
+import dynamic from 'next/dynamic';
+
+import { Market } from '@ui/components/pages/Fuse/FusePoolPage/MarketsList';
+import { FundButton } from '@ui/components/pages/Fuse/FusePoolPage/MarketsList/FundButton';
+import CaptionedStat from '@ui/components/shared/CaptionedStat';
+import ClaimAssetRewardsButton from '@ui/components/shared/ClaimAssetRewardsButton';
+import { useChartData } from '@ui/hooks/useChartData';
+import { useColors } from '@ui/hooks/useColors';
+import { MarketData } from '@ui/types/TokensDataMap';
+import { shortUsdFormatter } from '@ui/utils/bigUtils';
+import { FuseUtilizationChartOptions } from '@ui/utils/chartOptions';
+
+const Chart = dynamic(() => import('react-apexcharts'), { ssr: false });
+
+export const AdditionalInfo = ({
+  row,
+  rows,
+  comptrollerAddress,
+  supplyBalanceFiat,
+}: {
+  row: Row<Market>;
+  rows: Row<Market>[];
+  comptrollerAddress: string;
+  supplyBalanceFiat: number;
+}) => {
+  const asset: MarketData = row.original.market;
+  const assets: MarketData[] = rows.map((row) => row.original.market);
+  const assetUtilization =
+    asset.totalSupply.toString() === '0'
+      ? 0
+      : parseFloat(
+          // Use Max.min() to cap util at 100%
+          Math.min(
+            (Number(utils.formatUnits(asset.totalBorrow, 18)) /
+              Number(utils.formatUnits(asset.totalSupply, 18))) *
+              100,
+            100
+          ).toFixed(0)
+        );
+
+  const { data } = useChartData(asset.cToken);
+  const { cChart, cPage } = useColors();
+
+  return (
+    <Box>
+      <HStack justifyContent="flex-end">
+        <ClaimAssetRewardsButton poolAddress={comptrollerAddress} assetAddress={asset.cToken} />
+        <FundButton
+          mode={FundOperationMode.SUPPLY}
+          comptrollerAddress={comptrollerAddress}
+          assets={assets}
+          asset={asset}
+        />
+        <FundButton
+          mode={FundOperationMode.WITHDRAW}
+          comptrollerAddress={comptrollerAddress}
+          assets={assets}
+          asset={asset}
+          isDisabled={asset.supplyBalanceFiat === 0}
+        />
+        <FundButton
+          mode={FundOperationMode.BORROW}
+          comptrollerAddress={comptrollerAddress}
+          assets={assets}
+          asset={asset}
+          isDisabled={asset.isBorrowPaused || supplyBalanceFiat === 0}
+        />
+        <FundButton
+          mode={FundOperationMode.REPAY}
+          comptrollerAddress={comptrollerAddress}
+          assets={assets}
+          asset={asset}
+          isDisabled={asset.borrowBalanceFiat === 0}
+        />
+      </HStack>
+      <Grid
+        templateColumns={{
+          base: 'repeat(1, 1fr)',
+          lg: 'repeat(2, 1fr)',
+        }}
+        w="100%"
+        gap={4}
+        alignItems="flex-end"
+      >
+        <Box
+          height="200px"
```

Comment:
Seeing hardcoded values here, how does it look on mobile? Seem like some of the users and from the team really look at the UI in mobile often.

Reply:
it's graph height and it works both desktop and mobile

Reply:
Ah alright makes sense to give it a fixed height 👍